kp: 4                       # Proportional Gain (anglePControllerCallback())
max_dot_theta: 2.5          # Maximum allowed angular velocity (in rad/sec)
max_acc: 0.5                # Maximum allowed acceleration in the velocity smoother (avoid slippage)
t_max: 3.                   # Time constant for the velocity smoother
ks: 50.                     # Gain factor of the velocity smoother
angle_deadzone: 0.2         # DeadZone range (in rad/sec), values in this range are considered zero
max_vel: 0.7                #(rosparam_get max_vel) Maximum desired velocity
near_goal_distance: 1       # distance robot-tower below which near_goal condition is activated
proximity_threeshold: 0.75  # If an obstacle is sensed below this threeshold the navigation start to be performed by fuzzy obstacle avoidance
dontcare: 0.64              # Obstacle in proximity condition but if above this threeshold still don't affect the navigation trajectory